demo/settings/production.py

```diff
@@ -0,0 +1,15 @@
+import django_heroku
+from demo.settings.common import *
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ['SECRET_KEY']
```

Comment:
@camilamaia I am assuming the demo-api uses `os.environ['SECRET_KEY']` to get the secret key on Heroku.

Reply:
I was earlier reading the [blog](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/Deployment) on MDN that uses Heroku most of my assumptions are from that blog.